flickit-assessment-kit/src/main/java/org/flickit/assessment/kit/adapter/in/rest/subject/CreateSubjectRestController.java

```diff
@@ -0,0 +1,40 @@
+package org.flickit.assessment.kit.adapter.in.rest.subject;
+
+import lombok.RequiredArgsConstructor;
+import org.flickit.assessment.common.config.jwt.UserContext;
+import org.flickit.assessment.kit.application.port.in.subject.CreateSubjectUseCase;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+public class CreateSubjectRestController {
+
+    private final CreateSubjectUseCase useCase;
+    private final UserContext userContext;
+
+    @PostMapping("assessment-kits/{kitId}/subjects")
+    public ResponseEntity<CreateSubjectResponseDto> createSubject(@PathVariable("kitId") Long kitId,
+                                                                  @RequestBody CreateSubjectRequestDto createSubjectRequestDto) {
```

Comment:
You can rename `createSubjectRequestDto` to `request` or `requestDto` like other controllers.

Reply:
Do the same in `toParam` method parameters